### PR TITLE
 fix(layouts): resolving with nested custom dir

### PR DESCRIPTION
**Pull request by TheAlexLichter**
Layouts did not resolve properly when `dir.layouts` is a nested folder.
This PR solves the problem

## Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly. (PR: #)
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [x] All new and existing tests are passing.

### CODE REVIEW

**Review by clarkdo — APPROVED** [review]

File: lib/builder/builder.js [hunk 1]
```diff
@@ -254,9 +254,7 @@ export default class Builder {
       })
       layoutsFiles.forEach((file) => {
         const name = file
-          .split('/')
-          .slice(1)
-          .join('/')
+          .replace(`${this.options.dir.layouts}/`, '')
```
Comment (clarkdo) [inline lib/builder/builder.js]: maybe using regex /^ for more accurate matching?

**Review by clarkdo — APPROVED** [review]

**Review by pi0 — APPROVED** [review]

**Comment by ajcastro** [comment]
Hello my nuxt community!
I structured my nuxt app inside a `src` directory. I am just wondering if my `src\layouts` will work? 
Is there something should I do first?

**Comment by manniL** [comment]
@ajcastro That's fine. Only nested directories in relative to `srcDir` were affected.

**Comment by ajcastro** [comment]
thanks @manniL <3

**Pull request merged**